README.md

```diff
@@ -255,8 +255,7 @@ pact.verifyPacts({
 | `providerBaseUrl`           | true      | string  | Running API provider host endpoint.                                                                        |
 | `pactBrokerUrl`         | false     | string  | Base URL of the Pact Broker from which to retrieve the pacts. Required if `pactUrls` not given.                                                                        |
 | `provider`                  | false     | string  | Name of the provider if fetching from a Broker                                                             |
-| `tags`                      | false     | array   | Array of tags, used to filter pacts from the Broker                                                        |
-| `consumerVersionTag`        | false     | string  | Retrieve the latest pacts with this consumer version tag                                                        |
+| `consumerVersionTag`        | false     | string | array  | Retrieve the latest pacts with this consumer version tag(s)                                                        |
```

Comment:
The extra pipe denoting the union type here will actually try and create a new table column. Need to escape the `|` or problem just use `or` instead.